### PR TITLE
[ELY-1686] Add documentation for the new Utility to port legacy properties files into a FileSystemRealm

### PR DESCRIPTION
**Pull request by justinmcook**
Jira: https://issues.jboss.org/browse/ELY-1686

Requires https://github.com/wildfly-security/wildfly-elytron-tool/pull/161 to be merged

### CODE REVIEW

**Comment by justinmcook** [comment]
Still needs to be prechecked

**Comment by fjuma** [comment]
@justinmcook I know you made some updates a couple weeks ago to the implementation for this based on feedback from Martin. Do any of those changes require any updates to the documentation here?

File: docs/src/main/asciidoc/_elytron/migrate/Properties_File_Based_Authentication_Migration.adoc [hunk 1]
```diff
@@ -243,6 +243,104 @@ Which results in: -
 At this stage the authentication is the equivalent of the original
 configuration however now Elytron components are used exclusively.
 
+[[FileSystemRealm_Based_Authentication_Migration]]
+== Migrating to FileSystemRealm Based Authentication
+
+An alternative to using a properties-realm in Elytron is to use
```
Comment (fjuma) [inline docs/src/main/asciidoc/_elytron/migrate/Properties_File_Based_Authentication_Migration.adoc]: I think this paragraph should explicitly mention that this command can now be used to convert legacy properties files into an Elytron filesystem-realm.

Comment (justinmcook) [reply docs/src/main/asciidoc/_elytron/migrate/Properties_File_Based_Authentication_Migration.adoc]: Modified the paragraph

**Comment by justinmcook** [comment]
@fjuma Made some minor updates to the documentation to reflect updates

**Review by fjuma — APPROVED** [review]

**Pull request merged**